_projects/geotrellis-vt-server.md

```diff
@@ -0,0 +1,57 @@
+---
+# Remove the project attributes which don't apply to your project. (ie: if no issues, delete the issues section)
+
+published: false # change to true once ready to post
+
+title: "Dynamic Vector Tile Server" # project title inside quotes
+excerpt: "Serve dynamic vector tiles from GeoJSON and other sources using GeoTrellis" # shows on project list page
+
+# project attributes
+
+requirements: # bullet list of requirements – one requirement per line – follow below format
+  - "experience building or using APIs"
+  - "interest in Scala / FP"
+
+tags: # one tag per line – spaces are allowed in tags – use tags other posts use
+  - "vector tiles"
+  - "scala"
+  - "GeoTrellis"
+
+difficulty: "medium" # Easy, Medium, Hard – pick one
+
+mentors: # github username without the @ – example: designmatty
+  - "jisantuc"
+  - "cloudniner"
+  
+---
+
+# Motivation
+
+Prior art exists showing efficient vector tile slicing and rendering for static data. Raster Foundry and other projects have a need for the ability to serve dynamically generated vector tiles from data provided by users as they use the application. Several other options (t-rex, tegola, tilegarden) require configuration to serve a set of layers when the server starts, but the goal of this project is for clients to ad hoc specify layers.
```

Comment:
Might be implicit in the conceit, not sure what is the path of the reader to this document but might be worth while mention as motivation and expectation setting that this will be likely contribution to at least two open source projects, GeoTrellis and Raster Foundry.

Reply:
Ah good yeah -- "you get to contribute to open source" is something that we'd want people to be excited about, so we should tell them

Reply:
Also might not be known to the potential contributor why vector tiles are exciting, so mentioning that they enable advanced client side visualizations and linking to some intro/demo material could be useful here.